src/libraries/System.Configuration.ConfigurationManager/tests/System.Configuration.ConfigurationManager.Tests.csproj

```diff
@@ -97,9 +97,6 @@
   <ItemGroup>
     <ProjectReference Include="..\src\System.Configuration.ConfigurationManager.csproj" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'">
-      <ProjectReference Include="$(LibrariesProjectRoot)System.Diagnostics.TraceSource\src\System.Diagnostics.TraceSource.csproj" />
```

Comment:
@ViktorHofer would be nice if we had some enforcement of our reference rules to catch when folks are referencing things in wrong/unnecessary ways.

Reply:
This quick fix looks good but in general we want to support both ways of referencing a shared framework library. I hadn't had time to look into the actual issues more closely but I would have assumed that `System.Diagnostics.TraceSource.csproj`'s ProjectReferences aren't transitively passed to the test project because of https://github.com/dotnet/runtime/blob/e461b776bff0e5a6b8c72282a9f66695a837461d/eng/references.targets#L26-L29.